packages/templates/app/python/polywrap.graphql

```diff
@@ -0,0 +1 @@
+#import * into Ethereum from "ens/wraps.eth:ethereum@1.1.0"
```

Comment:
We should be using wrapscan.io URIs everywhere now. For this, use `wrapscan.io/polywrap/ethers-utils@1.0`

Reply:
"encode params" can come from the ethers-utils wrap, and does not require the ethereum-wallet plugin. I've pushed a change to fix this.